upload_files/README.rst

```diff
@@ -0,0 +1,122 @@
+edx-e2e-tests
```

Comment:
could you simplify this file to make it more obvious it's a test file? it might be confusing to someone that is traversing our filesystem.